models/user.model.js

```diff
@@ -35,6 +35,11 @@ const userSchema = new Schema(
       type: String,
       default: 'https://th.bing.com/th/id/OIP.z3fa8PjEnvzg4bhW61tEOwAAAA?rs=1&pid=ImgDetMain',
     },
+    role: {
+      type: String,
+      enum: ['admin', 'teacher', 'member'],
+      default: 'member',
+    },
```

Comment:
```suggestion
    role: {
      type: String,
      enum: USER_ROLE_ENUM,
      default: USER_ROLE_ENUM.MEMBER,
    },
```

move vào constant, gender cũng làm tưng tự
```
const USER_ROLE_ENUM = {
  ADMIN: 'admin',
  MEMBER: 'member',
  TEACHER: 'teacher',
};
```